web/ui/src/common/resourcesFactory.js

```diff
@@ -312,6 +312,88 @@
             }
         };
 
+        var v2MethodConfigs = {
+            getService: {
+                method: GET,
+                url: id => `/api/v2/services/${id}`,
+            },
+            updateService: {
+                method: PUT,
+                url: id => `/api/v2/services/${id}`,
+                payload: (id, service) => service
+            },
+            getServiceAncestors: {
+                method: GET,
+                url: id => `/api/v2/services/${id}?ancestors`,
+            },
+            getServiceChildren: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/services`,
+            },
+            getServiceConfig: {
+                method: GET,
+                url: id => `/api/v2/serviceconfigs/${id}`,
+            },
+            updateServiceConfig: {
+                method: PUT,
+                url: (id) => `/api/v2/serviceconfigs/${id}`,
+                payload: (id, cfg) => {return JSON.stringify({
+                    'Filename':    cfg.Filename, 
+                    'Owner':       cfg.Owner, 
+                    'Permissions': cfg.Permissions, 
+                    'Content':     cfg.Content
+                });}
+            },
+            getServiceConfigs: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/serviceconfigs`,
+            },
+            getServiceContext: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/context`,
+            },
+            updateServiceContext: {
+                method: PUT,
+                url: id => `/api/v2/services/${id}/context`,
+                payload: (id, ctx) => ctx
+            },
+            getServiceInstances: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/instances`,
+                ignorePending: true
+            },
+            getServiceIpAssignments: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/ipassignments?includeChildren`,
+            },
+            getServiceMonitoringProfile: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/monitoringprofile`,
+            },
+            getServicePublicEndpoints: {
+                method: GET,
+                url: id => `/api/v2/services/${id}/publicendpoints?includeChildren`,
+            },
+            getServiceExportEndpoints: {
```

Comment:
We could, but right now we always want all children to make sure it behaves like the old code did. When the need arises, we can easily add the second arg to choose not to get all children